charts/kong/templates/_helpers.tpl

```diff
@@ -255,12 +255,19 @@ Create KONG_STREAM_LISTEN string
     {{- $listenConfig := dict -}}
     {{- $listenConfig := merge $listenConfig . -}}
     {{- $_ := set $listenConfig "address" "0.0.0.0" -}}
+    {{/* You set NGINX stream listens to UDP using a parameter because reasons. Since our configuration
+         is dual-purpose for both the Service and listen string, we forcibly inject this parameter if
+         that's the Service protocol. The default handles configs that predate the addition of the
+         protocol field, where we only supported TCP. */}}
```

Comment:
I'm worried that the language here would potentially confuse some readers, how would you feel about this instead?
```suggestion
    {{/* You set NGINX stream listens to UDP using a parameter due to historical reasons.
         our configuration is dual-purpose for both the Service and listen string, we forcibly inject
         this parameter ifthat's the Service protocol. The default handles configs that predate the
         addition of theprotocol field, where we only supported TCP. */}}
```

Reply:
b2522aa6f as Github edit did unpleasant things to spacing.